Fitness/Views/ProBioPageViewController.swift

```diff
@@ -0,0 +1,301 @@
+        nameLabel.snp.makeConstraints{ make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(profilePicContainerView.snp.bottom).offset(-64)
+        }
+        
+        bioTitleLabel.snp.makeConstraints{ make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(profilePicContainerView.snp.bottom).offset(64)
+            make.leading.equalToSuperview().offset(40)
+            make.trailing.equalToSuperview().offset(-40)
+        }
+        
+        bioTextView.snp.makeConstraints{ make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(bioTitleLabel.snp.bottom).offset(44)
+            make.leading.equalToSuperview().offset(44)
+            make.trailing.equalToSuperview().offset(-44)
+        }
+        
+        bioExpertiseDivider.snp.makeConstraints{ make in
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(2)
+            make.top.equalTo(bioTextView.snp.bottom).offset(36)
+        }
+        
+        expertiseTitleLabel.snp.makeConstraints{ make in
+            make.top.equalTo(bioExpertiseDivider.snp.bottom).offset(23)
+            make.centerX.equalToSuperview()
+        }
+        
+        expertiseTextView.snp.makeConstraints{ make in
+            make.top.equalTo(expertiseTitleLabel.snp.bottom).offset(16)
+            make.leading.equalToSuperview().offset(55)
+            make.trailing.equalToSuperview().offset(-55)
+        }
+        
+        expertiseRoutinesDivider.snp.makeConstraints{ make in
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(2)
+            make.top.equalTo(expertiseTextView.snp.bottom).offset(23)
+        }
+        
+        routinesTitleLabel.snp.makeConstraints{ make in
+            make.leading.equalToSuperview().offset(33)
+            make.trailing.equalToSuperview().offset(-33)
+            make.top.equalTo(expertiseRoutinesDivider.snp.bottom).offset(24)
+        }
+        
+        routinesView.snp.makeConstraints{ make in
+            make.leading.equalToSuperview().offset(24)
+            make.trailing.equalToSuperview().offset(-24)
+            make.top.equalTo(routinesTitleLabel.snp.bottom).offset(17)
+        }
+        
+        linksTitleLabel.snp.makeConstraints{ make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(routinesView.snp.bottom).offset(61)
+        }
+        
+        linksCollectionView.snp.makeConstraints{ make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(linksTitleLabel.snp.bottom).offset(12)
+            make.height.equalTo(30)
+            make.width.equalTo(30 * pro.links.count + (pro.links.count - 1) * 6)
+            make.bottom.equalToSuperview().offset(-100)
+        }
+    }
+    
+        
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        if collectionView == self.linksCollectionView {
```

Comment:
Is there ever a case where it's not the linksCollectionView? Also self shouldn't be necessary here.

Reply:
I'll remove the self, but the if statement is in case other CollectionViews are added in the future